### PR TITLE
Sweeper initialization and local session

### PR DESCRIPTION
**Pull request by rayklauck**
- introduces a Sweeper with create method
 - introduces a local session, which may be used for the not-remote sweeper nodetree

### CODE REVIEW

**Comment by rayklauck** [comment]
@tobiasah please have a look on the code. (It is only a fraction of the current status)

File: src/labone/sweeper/local_session.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+    def sync_get(self, path: LabOneNodePath):
+        try:
+            return self._memory[path]
+        except KeyError as e:
+            raise SweeperLocalStateError(f"Trying to read sweeper path '{path}', which does not exist.") from e
+
+    def sync_set(self, value: AnnotatedValue):
+        if value.path not in self._memory:
+            raise SweeperLocalStateError(
+                f"Trying to write sweeper path '{value.path}', which does not exist.")
```
Comment (tobiasah) [inline src/labone/sweeper/local_session.py:41]: suggestion: Remove the if and just wrap the following code in a try catch block

File: src/labone/sweeper/sweeper_nodes.json
Comment (tobiasah) [inline src/labone/sweeper/sweeper_nodes.json:1]: issue: Move this to `labone. resources`

**Review by tobiasah — APPROVED** [review]
LGTM.
I would have split this into two commits. The first is introducing the local session and the second one the sweeper initialization.

**Pull request closed without merging**